contracts/mainnet/handlers/StableFactoryHandler.vy

```diff
@@ -279,8 +279,8 @@ def get_pool_from_lp_token(_lp_token: address) -> address:
 @external
 @view
 def get_pool_name(_pool: address) -> String[64]:
-    if self.base_registry.get_n_coins(_pool) == 0:
-        return ""
+    # if self.base_registry.get_n_coins(_pool) == 0:
+    #     return ""
```

Comment:
Since Stable Factory pools are ERC20 tokens, I am instead doing this:

```
@external
@view
def get_pool_name(_pool: address) -> String[64]:
    return ERC20(_pool).name()
```